iaso/api/org_unit_change_requests/serializers.py

```diff
@@ -213,8 +221,49 @@ def validate(self, attrs):
             "new_location",
             "new_reference_instances",
         ]
-        if not any([validated_data.get(field) for field in new_fields]):
+        if not any([attrs.get(field) for field in new_fields]):
```

Comment:
This change is a bit surprising.

Reply:
Yes this is strange… I'll revert this since I cannot explain it :D